script.js

```diff
@@ -122,6 +204,367 @@ const introData = intro.map((item) => {
+  div.className = elem.cls5;
+  const h3 = document.createElement("h3");
+  h3.innerHTML = elem.h3;
+  div.appendChild(h3);
+  const para = document.createElement("p");
+  para.innerHTML = elem.para;
+  div.appendChild(para);
+  const ul = document.createElement("ul");
+  elem.list.map((item) => {
+    const li = document.createElement("li");
+    const aTag = document.createElement("a");
+    aTag.innerHTML = item;
+    li.appendChild(aTag);
+    ul.appendChild(li);
+    div.appendChild(ul);
+  });
+  const btn = document.createElement("button");
+  btn.innerHTML = elem.btn;
+  div.appendChild(btn);
+  return div;
+});
+
+// about
+
+const abtData = about.map((elem) => {
+  const div = document.createElement("div");
+  div.className = elem.cls;
+  const h2 = document.createElement("h2");
+  h2.innerHTML = elem.h2;
+  div.appendChild(h2);
+  const para = document.createElement("p");
+  para.innerHTML = elem.para;
+  div.appendChild(para);
+  const btn = document.createElement("button");
+  btn.innerHTML = elem.btn;
+  div.appendChild(btn);
+  return div;
+});
+
+// skillset
+const sklSet = about.map((elem) => {
+  const div = document.createElement("div");
+  div.className = elem.cls1;
+  elem.cls2.map((e) => {
+    const div1 = document.createElement("div");
+    div1.className = e;
+    if (div1.className === "specifics one") {
+      const h3 = document.createElement("h3");
+      const span = document.createElement("span");
+      span.innerHTML = elem.entity;
+      h3.appendChild(span);
+      h3.innerHTML += elem.h3;
+      div1.appendChild(h3);
+      const ul = document.createElement("ul");
+      elem.list.map((item) => {
+        const li = document.createElement("li");
+        li.innerHTML = item;
+        ul.appendChild(li);
+        div1.appendChild(ul);
+      });
+    } else if (div1.className === "specifics two") {
+      const h3 = document.createElement("h3");
+      const span = document.createElement("span");
+      span.innerHTML = elem.entity1;
+      h3.appendChild(span);
+      h3.innerHTML += elem.h3;
+      div1.appendChild(h3);
+      const ul = document.createElement("ul");
+      elem.list1.map((item) => {
+        const li = document.createElement("li");
+        li.innerHTML = item;
+        ul.appendChild(li);
+        div1.appendChild(ul);
+      });
+    } else if (div1.className === "specifics three") {
+      const h3 = document.createElement("h3");
+      const span = document.createElement("span");
+      span.innerHTML = elem.entity2;
+      h3.appendChild(span);
+      h3.innerHTML += elem.h3;
+      div1.appendChild(h3);
+      const ul = document.createElement("ul");
+      elem.list2.map((item) => {
+        const li = document.createElement("li");
+        li.innerHTML = item;
+        ul.appendChild(li);
+        div1.appendChild(ul);
+      });
+    }
+
+    div.appendChild(div1);
+  });
+
+  return div;
+});
+
+// popUp
+
+const popData = popUp.map((elem) => {

```

Comment:
- [ ] @team kindly check on the design of the popup, the region outside of the popup should be blurred as in the design, the popup should fit the whole width it should leave some margins, and also the popup should be in a fixed position in your case when I scroll the popup moves with other page content. Below is a comparison:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/202844508-3c2ddc84-5c43-4c20-9891-6372658b2e11.png) |  ![image](https://user-images.githubusercontent.com/61408860/202844531-82e689c7-cbfc-4b47-b948-1e7f8c83e2d4.png)

##mobile view

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/202844605-84072233-dcb7-4eb1-a8a4-0b4e3969fcf8.png) |  ![image](https://user-images.githubusercontent.com/61408860/202844627-f3c35400-775e-4980-aa6c-05234e395878.png)